lib/debug/breakpoint.rb

```diff
@@ -283,14 +302,17 @@ def setup
 
     def to_s
       if defined? @prev
-        "watch bp: #{@expr} = #{@prev} -> #{@current}"
+        "#{LABEL} #{@expr} = #{@prev} -> #{@current}"
```

Comment:
btw I found that the debugger enters an infinite loop (deadlock) when I call any color helper inside a breakpoint instance. it's part of the reason I store the label in a constant.

Reply:
@ko1 do you know the possible cause of this issue? it doesn't matter what I colorize, `IRB::Color.colorize` seems to trigger a stop when called here.

<img width="788" alt="截圖 2021-06-03 下午10 30 06" src="https://user-images.githubusercontent.com/5079556/120661898-41fd9b80-c4bb-11eb-95e0-623c43d27035.png">

Reply:
Not sure why...
Using `rescue Exception` on this method to check the exception can help you.

Reply:
The cause is not the coloring helper. Calling any non-internal or non-debugger method like `pp` (I'll just call it "the outsider method" can also cause this issue. It happens like this:

1. A breakpoint is triggered by a state change (`@name = nil => John`)
2. The session would use `show_bps` to show breakpoints.
3. When `WatchExprBreakpoint#to_s` is called, it then calls the outsider method.
4. The outsider method will trigger `WatchExprBreakpoint`'s tracepoint call and then `watch_eval`.  
5. It'll then spot another state change (`@name = John => nil`) because `WatchExprBreakpoint` is class agnostic.
6. This triggers another breakpoint suspend call and back to the 2nd step. The infinite loops begins.

I guess this just provided another reason to drop the expression breakpoint.

Reply:
I see. Maybe it should be fixed, but now we can drop it for a while.